lms/djangoapps/grades/admin/models.py

```diff
@@ -0,0 +1,66 @@
+"""
+Models defining feature flag controlling persistent grades globally
+and for specific courses.
+"""
+from config_models.models import ConfigurationModel
+from xmodule_django.models import CourseKeyField
+
+
+class PersistentGradesEnabledFlag(ConfigurationModel):
+    """
+    Enables persistent grades across the platform.
+    When this feature flag is set to true, individual courses
+    must also have persistent grades enabled for the
+    feature to take effect.
+    """
+
+    @classmethod
+    def feature_enabled(cls, course_id=None):
+        """
+        Looks at the currently active configuration model to determine whether the subsection grades feature is
+        available.
+
+        If the flag is not enabled, the feature is not available.
+        If the flag is enabled and the provided course_id is authorized,
```

Comment:
Nit: I don't believe the word "authorized" is correct in this context.  This is for incrementally rolling out the feature, and it's more about "enabling" rather than "authorization".  But I won't block the PR for this.